src/main/scala/com/rdio/thor/BaseImageService.scala

```diff
@@ -63,24 +61,29 @@ abstract class BaseImageService(conf: Config, client: Service[Request, Response]
     res
   }
 
-  def requestImage(url: String): Future[Option[Image]] = {
-    val req = Request("/" + url)
+  def requestImage(url: URL): Future[Option[Image]] = {
```

Comment:
Using URL instead of String is problematic, because what if we wanted to use relative paths? This change makes the code less general, and breaks backwards-compatibility.